### PR TITLE
Fixed the voice call panel

### PR DESCRIPTION
**Pull request by MidyGamy**
so I had a display problem after installing the theme and noticed it was due to the "Show Voice Users" option of discord.
So I made this so the activity panel and actions button adapt to that list.

I also brought back the link to get back the vc channel.

Before (actions buttons don't shows up)
image

After
image

btw it also fixes an incomatibility with Vencord "Game Activity Toggle" plugin.

### CODE REVIEW

**Comment by MidyGamy** [comment]
Before (actions buttons don't shows up)
image

After
image

**Comment by MidyGamy** [comment]
btw it also fixes an incomatibility with Vencord "Game Activity Toggle" plugin

File: bd-scss.config.js
Comment (Gibbu) [inline bd-scss.config.js:1]: This doesn't need to be included in the PR.

File: package-lock.json
Comment (Gibbu) [inline package-lock.json:1]: This also doesn't need to included in the PR as the repo uses PNPM not npm.

File: src/theme/sidebar/_panels.scss [hunk 1]
```diff
@@ -139,19 +161,16 @@
...
 		border: none;
 		background: transparent;
 		padding: 8px 16px;
 		&:has(+ .wrapper_e832ee:not(:empty)) {
-			margin-bottom: 56px;
-			z-index: 1;
+			.wrapper_e832ee > & {
+				margin-bottom: calc(100% - 3px);
+			}
+			z-index: 0;
```
Comment (Gibbu) [inline src/theme/sidebar/_panels.scss:173]: Please move the top level rules before the children rules.
Example:
```scss
&:has(+ .wrapper_e832ee:not(:empty)) {
  z-index: 1;
  .wrapper_e832ee > & {
    margin-bottom: calc(100% - 3px);
  }
}
```

**Comment by MidyGamy** [comment]
so do I remove it and add it to the .gitignore?

**Review by Gibbu — CHANGES_REQUESTED** [review]
Please fix the issues and I'll gladly accept.

**Comment by Gibbu** [comment]
No, just use PNPM. It's much better in every way than npm.

File: .gitignore
Comment (Gibbu) [inline .gitignore:1]: Adding `bd-scss.config.js` to the gitignore file will make it so all future changes to the file will be ignored.

File: bd-scss.config.js
Comment (Gibbu) [inline bd-scss.config.js:1]: Removing the file from the PR and removing it from the repo are two very different things.
This is telling git to remove it from the repo (which we don't want).

**Review by Gibbu — CHANGES_REQUESTED** [review]

**Comment by MidyGamy** [comment]
ok so how do I tell the PNPM command to set the output to Vencord?

**Comment by Gibbu** [comment]
It's not an pnpm command, it's a node command. 
PNPM is the package manager that is used to download the `bd-scss` package and all it's dependencies, which also happenes to run node commands. You can stlll use `npm run dev` and change the bd-scss config file accordingly.

Don't don't add the `bd-scss.config.js` changes to the repo.

**Comment by MidyGamy** [comment]
ok

**Comment by MidyGamy** [comment]
should be good now

**Review by Gibbu — APPROVED** [review]

**Pull request merged**